hw/data/common_project_cfg.hjson

```diff
@@ -35,7 +35,15 @@
   revision: '''{eval_cmd}
             COMMIT_L=`git rev-parse HEAD`; \
             COMMIT_S=`git rev-parse --short HEAD`; \
-            echo "Revision: [\`$COMMIT_S\`]({results_server_url_prefix}{repo_server}/tree/$COMMIT_L)";
+            REV_STR="Revision: [\`$COMMIT_S\`]({results_server_url_prefix}{repo_server}/tree/$COMMIT_L)"
+            if [ -d "hw/foundry" ]; then \
+              cd hw/foundry; \
+              COMMIT_FOUNDRY_S=`git rev-parse --short HEAD`; \
+              REV_STR_FOUNDRY="Foundry Revision: \`$COMMIT_FOUNDRY_S\`"
```

Comment:
Anchor all paths with `{proj_root}`. Also, `git -C` allows you to set a different repo path.  

```suggestion
            if [ -d "{proj_root}/hw/foundry" ]; then \
              COMMIT_FOUNDRY_S=`git -C {proj_root}/hw/foundry rev-parse --short HEAD`; \
              REV_STR_FOUNDRY="Foundry Revision: \`$COMMIT_FOUNDRY_S\`" \
```

Reply:
makes sense, thanks